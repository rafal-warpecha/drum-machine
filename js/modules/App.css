.container {
    text-align: center;
    font-size: 72px;
}
