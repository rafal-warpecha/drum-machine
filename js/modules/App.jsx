import React, { Component } from 'react';
import styles from './App.css';

export default class App extends Component {
    componentWillMount() {

    }

    render() {
        return (
            <div className={styles.container}>
                <span>App Name</span>
            </div>
        );
    }
}
