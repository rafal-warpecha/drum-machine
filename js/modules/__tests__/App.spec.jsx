import React from 'react';
import { shallow } from 'enzyme';
import App from './../App';

test('App match snapshot', () => {
    const component = shallow(<App />);

    expect(component).toMatchSnapshot();
});
