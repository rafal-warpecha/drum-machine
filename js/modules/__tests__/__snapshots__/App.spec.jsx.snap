// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App match snapshot 1`] = `
<div
  className="container"
>
  <span>
    App Name
  </span>
</div>
`;
